packages/codegen/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "oazapfts",
+  "version": "0.0.0-development",
+  "description": "OpenApi TypeScript client generator",
+  "license": "MIT",
+  "repository": "oazapfts/oazapfts",
```

it's nice to include a `directory` here for monorepos
```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/oazapfts/oazapfts",
    "directory": "packages/codegen"
  },
```